primefaces-showcase/src/main/webapp/ui/df/nested.xhtml

```diff
@@ -53,4 +53,16 @@
 
     </ui:define>
 
+    <ui:define name="more-source-tabs">
+        <p:tab title="/ui/df/level1.xhtml">
+            <f:attribute name="flatten" value="true"/>
+        </p:tab>
+        <p:tab title="/ui/df/level2.xhtml">
+            <f:attribute name="flatten" value="true"/>
+        </p:tab>
+        <p:tab title="/ui/df/level3.xhtml">
+            <f:attribute name="flatten" value="true"/>
```

Comment:
You might want to check here I had to set flatten="false" on your others as it duplicated many of the Java tabs twice.